cirq/linalg/decompositions_perf_test.py

```diff
@@ -30,3 +30,4 @@
 def test_kak_decomposition_perf(target, benchmark):
     kak = benchmark(cirq.kak_decomposition, target)
     np.testing.assert_allclose(cirq.unitary(kak), target, atol=1e-8)
+# yapf: enable
```

Comment:
Do you need yapf disabled from lines 19 to 32?

Reply:
Nope. Fixed.